python/cudf/cudf/core/column/categorical.py

```diff
@@ -964,6 +964,20 @@ def to_arrow(self):
             dictionary=self.categories.to_arrow(),
         )
 
+    @property
+    def values_host(self):
+        """
+        Return a numpy representation of the CategoricalColumn.
+        """
+        return self.to_pandas().values
+
+    @property
+    def values(self):
+        """
+        Return a CuPy representation of the CategoricalColumn.
+        """
+        raise TypeError("Data must be numeric")
```

Comment:
Given `values_host` is going to return a `pd.Categorical` object, I think we should raise a `NotImplementedError` here instead that `cudf.Categorical` isn't implemented yet.